hi-ml-histopathology/src/histopathology/scripts/generate_crossval_report.py

```diff
@@ -93,7 +154,16 @@ def render_training_curves(report: HTMLReport, heading: str, level: int,
 def render_metrics_table(report: HTMLReport, heading: str, level: int,
                          metrics_df: pd.DataFrame, best_epochs: Optional[Dict[int, int]],
                          base_metrics_list: List[str], metrics_prefix: str) -> None:
-
+    """
```

Comment:
We're missing a docstring entry for `metrics_df` here!

Reply:
Well spotted, done now